arm/.global/global.module.tests.ps1

```diff
@@ -451,7 +451,12 @@ Describe 'Readme tests' -Tag Readme {
             $fileHashAfter = (Get-FileHash $readMeFilePath).Hash
 
             # Compare
-            $fileHashBefore -eq $fileHashAfter | Should -Be $true -Because 'The file hashes before and after applying the Set-ModuleReadMe function should be identical'
+            $filesAreTheSame = $fileHashBefore -eq $fileHashAfter
```

Comment:
Is this related or should we add it to the global testing PR?

Reply:
I have this now already in 3 different PRs as I need it to be able to troubleshoot the ReadMe generation. So yes - it is also in that one. ;) It's just an adjustment to the output as before it was really hard to say which readme generation failed if you have it running for all 85

Reply:
I'm not discussing the value of it, but please, especially if included in another PR, remove this change from here.

Reply:
It's not included. It was just something to make this PR work - and others after. Will create another PR for it

Reply:
Removed